### PR TITLE
Use the filesB64 field instead of the hardwired file fields

### PR DESCRIPTION
**Pull request by dfellis**
Part of the clean up and unification of the two deployment paths. This changes only the way they interact with the deploy API, not how they are built on each other (that will come later)

### CODE REVIEW

**Comment by dfellis** [comment]
Confirmation that arbitrary files can be uploaded via `alan deploy`:

Screenshot from 2021-07-01 13-16-32

Screenshot from 2021-07-01 13-16-47

**Comment by dfellis** [comment]
Also confirmed that `anycloud new` also works, but there's nothing there to screenshot to tell the difference.

**Review by cdmistman — APPROVED** [review]

File: avm/src/main.rs [hunk 1]
```diff
@@ -167,10 +170,17 @@ fn main() {
             let agz_file = matches.value_of("AGZ_FILE").unwrap();
             let app_name = matches.value_of("app-name").map(String::from);
             let config_name = matches.value_of("config-name").map(String::from);
+            let files = matches.value_of("files");
+            let mut files_b64 = HashMap::new();
+            if let Some(files) = files {
+              let names = files.split(",");
+              for name in names {
+                files_b64.insert(name.to_string(), get_agz_file_b64(name.to_string()).await);
```
Comment (aguillenv) [inline avm/src/main.rs:178]: Are you sure the `get_agz_file_b64` fn works for this?

File: avm/src/main.rs [hunk 2]
```diff
@@ -167,10 +170,17 @@ fn main() {
             let agz_file = matches.value_of("AGZ_FILE").unwrap();
             let app_name = matches.value_of("app-name").map(String::from);
             let config_name = matches.value_of("config-name").map(String::from);
+            let files = matches.value_of("files");
+            let mut files_b64 = HashMap::new();
+            if let Some(files) = files {
+              let names = files.split(",");
```
Comment (aguillenv) [inline avm/src/main.rs:176]: I was going to suggest if a `trim` would be necessary, but one cannot add spaces on bash command, right?

**Review by depombo — APPROVED** [review]

File: avm/src/main.rs (hunk 1 above)
Comment (dfellis) [reply avm/src/main.rs:178]: Yep! That's what my second screenshot was about: to show that it decoded just fine.

The name of that function isn't great, but it works perfectly.

File: avm/src/main.rs (hunk 2 above)
Comment (dfellis) [reply avm/src/main.rs:176]: It's theoretically possible but you have to intentionally bypass bash's standard parsing with backslashes or quotes.

**Review by aguillenv — APPROVED** [review]

**Pull request merged**